seaborn/categorical.py

```diff
@@ -2810,11 +2810,11 @@ def stripplot(x=None, y=None, hue=None, data=None, order=None, hue_order=None,
     {categorical_data}
     {order_vars}
     jitter : float, ``True``/``1`` is special-cased, optional
-        Amount of jitter (only along the categorical axis) to apply. This
-        can be useful when you have many points and they overlap, so that
-        it is easier to see the distribution. You can specify the amount
-        of jitter (half the width of the uniform random variable support),
-        or just use ``True`` for a good default.
+        Amount of jitter (only along the categorical axis) to apply. Defaults
```

Comment:
Doc string parameter information (generally) should not specify the default value. The defaults are apparent in the function signature, and defaults specified in the doc string test are easy to forget to update if the values change, making the documentation wrong.

Reply:
Thanks! Reverted the jitter docstring to its previous description.